packages/perseus-core/src/parse-perseus-json/perseus-parsers/hint.ts

```diff
@@ -17,6 +17,7 @@ export const parseHint: Parser<Hint> = object({
     replace: optional(boolean),
     content: string,
     widgets: defaulted(parseWidgetsMap, () => ({})),
-    metadata: optional(array(string)),
     images: parseImages,
+    // deprecated
+    metadata: any,
```

Comment:
We can't just ignore it? It has to be parsed?

Reply:
This is probably way too paranoid, but I want to avoid the situation where

- we delete `metadata` from the schema and parser.
- 5 years from now, someone decides it would be cool if `PerseusRenderer` had a `metadata` field.
- they get errors in production because legacy content already has `metadata`, and it's shaped totally differently from whatever new metadata format they came up with.

I guess the parser regression tests should stop them before they get to step 3, though?